src/js/facility-locator/reducers/facilities.js

```diff
@@ -0,0 +1,10 @@
+const INITIAL_STATE = { facilities: [], facilityDetail: null };
+
+export default function (state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case 'FETCH_VA_FACILITY':
+      return Object.assign({}, state, { facilityDetail: action.payload });
```

Comment:
Alternatively, you can use the ES6 spread operator:

`return { ...state, facilityDetail: action.payload };`

http://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html